src/main/java/edu/ucsb/cs156/courses/documents/ConvertedSection.java

```diff
@@ -0,0 +1,15 @@
+package edu.ucsb.cs156.courses.documents;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ConvertedSection {
+    CourseInfo courseInfo;
+    Section section;
```

Comment:
Can we add explicit access modifiers here?

Reply:
Fixed